libkbfs/tlf_journal.go

```diff
@@ -346,6 +379,13 @@ const (
 // enqueued journal ID tag.
 const CtxJournalOpID = "JID"
 
+func (j *tlfJournal) pause(pauseType tlfJournalPauseType) {
```

Comment:
I think this should be combined with `pauseBackgroundWork`, and `j.wg.Pause()` can be moved back to where it was. Then the check above the `1) Idle.` case in `doBackgroundWorkLoop` can be removed, since we're now pausing explicitly from CR.